libraries/chain/proposal_evaluator.cpp

```diff
@@ -23,17 +23,73 @@
  */
 #include <graphene/chain/proposal_evaluator.hpp>
 #include <graphene/chain/proposal_object.hpp>
-#include <graphene/chain/account_object.hpp>
-#include <graphene/chain/protocol/fee_schedule.hpp>
-#include <graphene/chain/exceptions.hpp>
-#include <graphene/chain/asset_evaluator.hpp>
-#include <graphene/chain/hardfork.hpp>
```

Comment:
IMHO `hardfork.hpp` should be kept here (after done the cleanup in evaluator.hpp`).